man/mount.efs.8

```diff
@@ -87,6 +99,20 @@ Mount an EFS file system with file system ID "fs-abcd1234" at mount point \
 sudo mount -t efs -o tls,verify=0 fs-abcd1234 /mnt/efs
 Mount an EFS file system with file system ID "fs-abcd1234" at mount point \
 "/mnt/efs" using encryption of data in transit and a verify level of 0\&.
+.TP
+sudo mount -t efs fs-abcd1234.efs.us-east-1.amazonaws.com /mnt/efs
```

Comment:
Having this FQDN option complicates the user experience of this new CNAME feature without an obvious benefit. Unless there is a strong reason to have this please limit the PR to include just the addition of the CNAME feature.